akka-http2-support/src/main/scala/akka/http/scaladsl/model/http2/Http2StreamIdHeader.scala

```diff
@@ -6,6 +6,7 @@ package akka.http.scaladsl.model.http2
 
 import akka.http.impl.engine.ws.InternalCustomHeader
 
+// TODO should the streamId not be a Long? I may be mis-remembering the spec
```

Comment:
I thought so too... basically I think we are having to do with Unsigned 4 octet long numbers. These don't have good representation in Scala world... as the '>>' vs. '>>>' shows. This goes for all 32-bit stuff extracted from the frames, so not only streamId.